pyiron_potentialfit/mlip/mlip.py

```diff
@@ -260,7 +260,7 @@ def _read_cfg(self, filename):
         store.add_array("stress", dtype=np.float64, shape=(6,), per="chunk")
         for cfg in loadcfgs(os.path.join(self.working_directory, filename)):
             struct = Atoms(
-                symbols=species[np.cast[np.int64](cfg.types)],
+                symbols=species[np.asarray(cfg.types, dtype=np.int64)],
```

Comment:
I used for my testing, but this should be identical :)
```suggestion
                symbols=species[cfg.types.astype(np.int64)],
```